with-router-uniwind/src/uniwind-types.d.ts

```diff
@@ -0,0 +1,10 @@
+// NOTE: This file is generated by uniwind and it should not be edited manually.
```

Comment:
If Uniwind generates this file, do we need it in the example?

Reply:
It won't be generated on `npx create expo`, but it should be when running `expo start` [by Uniwind's Metro config](https://github.com/Unistyles-OSS/uniwind/blob/0d1f562fdd9338cc3085e6bcbd2457b32b549dda/packages/uniwind/src/metro/injectThemes.ts#L39).